core/integration/file_test.go

```diff
@@ -112,6 +112,58 @@ func TestFileSize(t *testing.T) {
 	require.Equal(t, len("some-content"), res.Directory.WithNewFile.File.Size)
 }
 
+func TestFileName(t *testing.T) {
+	t.Parallel()
+
+	wd := t.TempDir()
+
+	c, ctx := connect(t, dagger.WithWorkdir(wd))
+
+	t.Run("new file", func(t *testing.T) {
```

Comment:
Can add `t.Parallel()` to these `t.Run` right?

Reply:
:tada: yup we can!